conftest.py

```diff
@@ -0,0 +1,28 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This file contains pytest configuration settings that are astropy-specific
+(i.e.  those that would not necessarily be shared by affiliated packages
+making use of astropy's test runner).
```

Comment:
I'm a bit worried that having this at the top-level will break running tests via:

```
import astropy
astropy.test()
```

since it presumably doesn't get installed?

Reply:
Yes, I think you're right. I honestly forgot to consider that use case since I thought using a top-level `conftest.py` was a little cleaner. But we do actually need the other plugins to be installed in order for the `astropy.test()` case to work, so I will move them back to `setup.py`.

Reply:
@astrofrog I found a way to allow `astropy.test` to make use of the internal plugins while still allowing for the use of a top-level `conftest.py` rather than registering the plugins in `setup.py`. In addition to being a cleaner solution (imo) this also has the benefit of avoiding some warnings that would otherwise result when running tests.